infrastructure/variables.tf

```diff
@@ -116,11 +116,11 @@ variable "max_clients" {
 }
 
 variable "scale_up_threshold" {
-  default = "40"
+  default = "1"
 }
 
 variable "scale_down_threshold" {
-  default = "10"
+  default = "0"
```

Comment:
I think this will require a corresponding change to `nomad_queue_length_alarm_down`:

```git 
- comparison_operator = "LessThanThreshold"
+ comparison_operator = "LessThanOrEqualToThreshold"
```